package.json

```diff
@@ -18,18 +18,20 @@
     "David Hemphill <davidlee.hemphill@gmail.com>"
   ],
   "scripts": {
-    "prebabelify": "rimraf lib",
+    "prebabelify": "npm run generate:plugin-list && rimraf lib",
     "babelify": "babel src --out-dir lib --copy-files",
     "postbabelify": "ncc build lib/cli-peer-dependencies.js -o peers",
     "rebuild-fixtures": "npm run babelify && babel-node scripts/rebuildFixtures.js",
     "prepublishOnly": "npm install --force && npm run babelify && babel-node scripts/build.js && node scripts/build-plugins.js",
     "style": "eslint .",
     "test": "cross-env TAILWIND_MODE=build jest",
     "test:integrations": "npm run test --prefix ./integrations",
+    "install": "npm run generate:plugin-list",
```

Comment:
```suggestion
    "postinstall": "npm run generate:plugin-list",
```

Reply:
Is there a difference?

Reply:
From npm docs:
> Don't use install. Use a .gyp file for compilation, and prepublish for anything else. You should almost never have to explicitly set a preinstall or install script.

However in this case we are using Babel that needs to be installed, so `postinstall` is the way to go.

Reply:
`Install` runs after installation (so babel will be installed), but before `postinstall`. I did find this in the docs though, so
> If there is a binding.gyp file in the root of your package and you haven't defined your own install or preinstall scripts, npm will default the install command to compile using node-gyp via node-gyp rebuild

[reference](https://docs.npmjs.com/cli/v7/using-npm/scripts#npm-install)

So while this doesn't apply to this repository, it does seem like there's some special behavior regarding `install` and `postinstall` is the safer option (combined with the part of the docs that you cited).